tom_observations/facilities/lco.py

```diff
@@ -311,6 +422,50 @@ def _build_configuration(self):
             }
         }
 
+    def _build_ephemeris_request_parts(self):
+        (new_targets, new_windows) = self._build_target_fields()
+        sites = new_targets.keys()
+        configurations = []
+        windows = []
+        locations = []
+        for site in sites:
+            for i in range(len(new_targets[site])):
+                single_obs_config = {
+                    'type': self.instrument_to_type(self.cleaned_data['instrument_type']),
+                    'instrument_type': self.cleaned_data['instrument_type'],
+                    'target': new_targets[site][i],
+                    'instrument_configs': [self._build_instrument_config()],
+                    'acquisition_config': {
+
+                    },
+                    'guiding_config': {
+
+                    },
+                    'constraints': {
+                        'max_airmass': self.cleaned_data['max_airmass']
+                    }
+                }
+                single_location = {'site': site,
+                                   'telescope_class': self._get_instruments()[self.cleaned_data['instrument_type']]['class']}
+                single_windows = [{'start': new_windows[site][i][0],
+                                   'end': new_windows[site][i][1]}
+                                  ]
+
+                configurations.append(single_obs_config)
+                windows.append(single_windows)
+                locations.append(single_location)
+        return (configurations, windows, locations)
+
+    def _build_ephemeris_requests(self):
+        (configurations, windows, locations) = self._build_ephemeris_request_parts()
+        requests = []
```

Comment:
requests shadows the package name. Scope should make it safe, but again not good form.

Reply:
Sorry not sure what you mean here. Can you clarify?

Reply:
Oh I think I see what you mean. I just tried to follow the style of some other parts in the code. Should I change the requests variable name, or see if they'd accept as is?